samples/suzieq-cfg.yml

```
@@ -11,34 +10,72 @@ rest:
   # rest_keyfile: /suzieq/key.pem
   #
   API_KEY: 496157e6e869ef7f3d6ecb24a6f6d847b224ee4f
-  address: 0.0.0.0
-  logfile: /tmp/sq-rest-server.log
   logging-level: WARNING
-  no-https: True # required for single HTTPS reverse proxy via Caddy
+  address: 127.0.0.1
```

```suggestion
  address: 0.0.0.0
```
Otherwise the container won't listen for request from the outside